docs/source/algorithms/LoadRaw-v3.rst

```diff
@@ -97,8 +97,8 @@ Output:
 
   Workspace has 2890 spectra
   Is detector 1001 a monitor? True
-  Workspace contains 35 logs
-  Workspace has property INT1: True
+  Workspace contains 33 logs
```

Comment:
This does not make any sense. Why did the numbers of logs change?

Reply:
What makes even less sense is that switching back to `master` gets 33 again.